### ISSUE TITLE
Best bet request for <WVPM locator>

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#98175]
# Template Instructions
You are here to create or change a Best Bet on VA.gov. A [best bet](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/standards/search-best-bets.md) enables a designated link to show up as a 'top recommendation' on va.gov/search/ . Not all links have a best bet -- the CAIA IA team has a set of criteria to apply when determining best bets, so we might suggest a different way to elevate your page! 

### 1. Select Request Type:

- [ ] NEW: You would like to create a new best bet
- [ ] CHANGE: You would like to change an existing best bet

### 2. Which page do you want to set up a best bet for?
What is the URL of your page? You can list multiple in one ticket.

https://www.womenshealth.va.gov/WOMENSHEALTH/wvpm-locator.asp

### 3. Why do you want to set up or change a best bet?
Include any research or data you have to support the request. 

We met with the Office of Women's Health who explained that VA received a Congressional request to address linking women Veterans to the facility and Women’s Veteran Program Manager that is closest to them. They've created this directory and we said we sould add crosslinks to it on relevant pages and could create a best bet (with Mikki's approval).

### 4. Any timing we should be aware of?
 Is the page currently live? If not, when is it launching? Do you have any target dates we should be aware of?

Because this was a congressional request the sooner the better. Mikki is aware and has given her approval already.

### 5. Notify CAIA
Post a link to this issue in Slack Channel #sitewide-content-ia and tag @Kristin Ouellette Muskat

 

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

***Note: All requests will be reviewed against the [Search best bet standards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/standards/search-best-bets.md) by the Sitewide Content, Accessibility, and IA team. Reach out via Slack #sitewide-content-ia with any questions.***

**Comment by kristinoletmuskat** [comment]
Hey @RLHecht , I just added this best bet. I'm trying to maximize its visibility, and it's competing with this one: https://www.va.gov/womenvet/

Our general guidance for best bets is to not have them for legacy sites, because we want to promote the modernized content. Do you know if that link above is important enough to have as a best bet? I can look into this, but wanted to see if you were familar with the page.

**Comment by RLHecht** [comment]
I know we have that site linked in the footer. And I think this directory is going to be linked to from that page as well. But I don't know how to make the call on whether it should be a best bet.

**Comment by RLHecht** [comment]
We told this group we would set up a best bet for the directory since it's part of a congressional request. It would also be one less click than getting to it from the women Veterans page. But I'm not sure what we do about the 2 competing with each other.

**Comment by kristinoletmuskat** [comment]
Yes! Sorry my message was confusing -- I did set up the directory best bet, and then I was trying to determine if we should take down the CWV. And I think we actually don't need to take down CWV.
